Source/JavaScriptCore/runtime/JSGenericTypedArrayView.h

```diff
@@ -435,13 +436,19 @@ inline RefPtr<typename Adaptor::ViewType> toUnsharedNativeTypedView(VM& vm, JSVa
 template<typename Adaptor>
 RefPtr<typename Adaptor::ViewType> JSGenericTypedArrayView<Adaptor>::toWrapped(VM& vm, JSValue value)
 {
-    return JSC::toUnsharedNativeTypedView<Adaptor>(vm, value);
+    auto result = JSC::toUnsharedNativeTypedView<Adaptor>(vm, value);
+    if (!result || result->isResizableOrGrowableShared())
+        return nullptr;
+    return result;
 }
 
 template<typename Adaptor>
 RefPtr<typename Adaptor::ViewType> JSGenericTypedArrayView<Adaptor>::toWrappedAllowShared(VM& vm, JSValue value)
 {
-    return JSC::toPossiblySharedNativeTypedView<Adaptor>(vm, value);
+    auto result = JSC::toPossiblySharedNativeTypedView<Adaptor>(vm, value);
+    if (!result || result->isResizableOrGrowableShared())
```

Comment:
Ditto: `AllowShared` vs rejecting `isResizableOrGrowableShared`?

Reply:
Ditto. This is correct.